File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/colors/MountainColor.java
package com.bonepl.chromaleague.hud.colors;

import com.bonepl.razersdk.color.Color;
import com.bonepl.razersdk.color.MultiTransitionColor;
import com.bonepl.razersdk.color.StaticColor;

import java.security.SecureRandom;

public class MountainColor implements Color {
    MultiTransitionColor mountainColor = new MultiTransitionColor.Builder(new StaticColor(8,3,0))
            .addTransition(StaticColor.BROWN, 10)
            .addTransition(new StaticColor(8,3,0), 10)
            .addTransition(new StaticColor(64,24,0),4)
            .looped(4)
            .build();

    public MountainColor() {
        int random = new SecureRandom().nextInt(mountainColor.getTotalTransitions());
        for (int i = 0; i < random; i++) {
            mountainColor.getColor();
        }
    }

    @Override
    public StaticColor getColor() {
        return mountainColor.getColor();
    }
}

File: src/hud/colors/MountainColor.ts
import {Color} from '../../razer-sdk/color/Color.js';
import {MultiTransitionColor} from '../../razer-sdk/color/MultiTransitionColor.js';
import {StaticColor} from '../../razer-sdk/color/StaticColor.js';

export class MountainColor implements Color {
    readonly mountainColor = new MultiTransitionColor.Builder(new StaticColor(8, 3, 0))
        .addTransition(StaticColor.BROWN, 10)
        .addTransition(new StaticColor(8, 3, 0), 10)
        .addTransition(new StaticColor(64, 24, 0), 4)
        .looped(4)
        .build();

    constructor() {
        const random = Math.trunc(Math.random() * this.mountainColor.getTotalTransitions());
        for (let i = 0; i < random; i++) {
            this.mountainColor.getColor();
        }
    }

    getColor(): StaticColor {
        return this.mountainColor.getColor();
    }
}
